### PR TITLE
update status of disnake http version

### PR DESCRIPTION
**Pull request by onerandomusername**
https://github.com/DisnakeDev/disnake/commit/2e5488572cc25432101da3328f44ae638d05dae6

### CODE REVIEW

File: libs.ts [hunk 1]
```diff
@@ -699,7 +699,7 @@ export const libs: Lib[] = [
 		name: 'disnake',
 		url: 'https://github.com/DisnakeDev/disnake',
 		language: 'Python',
-		apiVer: 8,
+		apiVer: 9,
```
Comment (advaith1) [inline libs.ts:702]: looks like this change isn't released yet?
```suggestion
		apiVer: '8 stable, 9 dev',
```

Comment (Koumes2) [reply libs.ts]: Pycord isn't using v9 in the stable version (they don't have anything in stable but anyway) and yet it's been mentioned as "stable". So, would the same logic apply to api version here?

Comment (onerandomusername) [reply libs.ts]: > Pycord isn't using v9 in the stable version (they don't have anything in stable but anyway) and yet it's been mentioned as "stable". So, would the same logic apply to api version here?

Probably worth making an issue, I didn't see this until you pointed it out.

Comment (advaith1) [reply libs.ts]: if there aren't visible stable and dev versions then the version that exists is considered stable

**Pull request merged**